compio-driver/src/iour/op.rs

```diff
@@ -475,3 +475,29 @@ impl<T: IoVectoredBuf> IntoInner for SendToVectored<T> {
         self.buffer
     }
 }
+
+/// A custom io-uring operation.
+pub struct IoUringOp {
```

Comment:
Call it custom op or something. IoUringOp in mod iour is repeating itself.

Reply:
I need to keep the struct name unchanged while `polling` feature enabled or not. If call it custom op here, it will be ambiguous in fusion mod.

Reply:
iour::CustomOp doesn't look ambiguous to me

Reply:
It may be ambiguous to create a `compio_driver::op::CustomOp`.